src/pudl/output/ferc1.py

```diff
@@ -10,13 +12,52 @@
 from matplotlib import pyplot as plt
 from networkx.drawing.nx_agraph import graphviz_layout
 from pandas._libs.missing import NAType as pandas_NAType
-from pydantic import BaseModel, validator
+from pydantic import BaseModel, confloat, validator
 
 import pudl
 
 logger = pudl.logging_helpers.get_logger(__name__)
 
 
+class CalculationToleranceFerc1(BaseModel):
```

Comment:
One of the things I was tracking in here originally went away, but I still think we are going to end up with a variety of metrics we want to check in the intra- and inter-table calculations, and the root-to-leaf calculations and having it be codified in a structured way seemed like it would be helpful.